exporters/otlp/otlp_exporter.cc

```diff
@@ -0,0 +1,71 @@
+#include "otlp_exporter.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace otlp
+{
+
+const std::string KCollectorAddress = "localhost:55678";
+
+// ----------------------------- Helper functions ------------------------------
+
+// Add span protobufs contained in recordables to request
+void PopulateRequest(const nostd::span<std::unique_ptr<sdk::trace::Recordable>> &spans,
+                     proto::collector::trace::v1::ExportTraceServiceRequest *request)
+{
+  auto resource_span = request->add_resource_spans();
+  auto instrumentation_lib = resource_span->add_instrumentation_library_spans();
+
+  for (auto &recordable : spans) {
+    auto rec = std::unique_ptr<Recordable>(
+      static_cast<Recordable *>(recordable.release()));
+
+    proto::trace::v1::Span* span = instrumentation_lib->add_spans();
```

Comment:
I wonder if you could do something like this here:
```c
instrumentation_lib->mutable_spans()->Add(std::move(rec->span()));
```
This might avoid the copy operation.

Reply:
I changed the code to use `std::move()` instead of `CopyFrom()`.